modules/game_interface/interface.otmod

```diff
@@ -36,5 +37,7 @@ Module
     - game_unjustifiedpoints
     - game_shaders
     - game_attachedeffects
+    - client_profiles
+    - game_bot
```

Comment:
```suggestion
    - game_bot
    - game_itemselector
```

Reply:
fix click on this
![image](https://github.com/mehah/otclient/assets/114332266/3b1b1a98-cc65-4a6b-b37c-e2761a9fc95c)